workbench-core/accounts/src/models/accounts/listAcountsRequest.ts

```diff
@@ -0,0 +1,30 @@
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const ListAccountsRequestParser = z.object({
```

Comment:
Can we add a test for this? It should throw `Boom.badInput` exceptions, and the message should be customer friendly.

Reply:
We don't want to throw the exception in here right? We want Zod to give us back all the errors together so that we can combine them in the error response, that way a user can fix everything at once and not one per response. The test needs to be at the controller/router level (which is the place that should be throwing the error) since this is a validation check. Its my understanding that we do controller/router level tests at the integration level, so I was leaving those for that.

Reply:
Ah got it. Let's get all the errors and provide one message for the requester.